core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -24,8 +24,14 @@ oppia.controller('Dashboard', [
       $scope, $rootScope, $window, oppiaDatetimeFormatter, alertsService,
       DashboardBackendApiService, RatingComputationService,
       ExplorationCreationService, FATAL_ERROR_CODES, UrlInterpolationService) {
+    var EXP_PUBLISH_TEXTS = {
+      default: 'This exploration is private. Publish it to receive statistics.',
+      sm: 'Publish the exploration to receive statistics.'
```

Comment:
Yes, the default message is too long on mobile. We can use a common message, it totally depends on the implementation. I had used different versions since the mocks show different versions in desktop and mobile versions.
What do you suggest ?